src/ralph/security/views.py

```diff
@@ -31,16 +43,19 @@ def scan_status(self, obj):
             html = self._to_span('', 'No scan')
         else:
             if scan.is_ok:
-                if obj.vulnerabilities_count > 0:
+                if not scan.is_patched:
                     html = self._to_span(
                         "alert", "Got vulnerabilities: {}".format(
-                            obj.vulnerabilities_count
+                            obj.exceeded_vulnerabilities_count
                         )
                     )
                 else:
                     html = self._to_span("success", "Host clean")
             else:
                 html = self._to_span("warning", "Scan failed")
```

Comment:
where do you find these classes (success, warning) for span? It's not working right now (afair it should be red when host has vulnerabilities)?